docs/pages/client_usage.mdx

```diff
@@ -0,0 +1,261 @@
+
+Contact your federation administrator to get a token for downloading a protected object. For OSDF users, contact [support@osg-htc.org](mailto:support@osg-htc.org) for help.
+
+Once you have the token, copy and paste it into a file on your machine, and pass the path to the file containing the token to the Pelican client with `-t` flag.
+
+```bash
+# Pass the path to the file containing the token using -t flag
+pelican object get pelican://<federation-url></namespace-prefix></path/to/file> </local/path/to/file> -t </path/to/token/file>
+```
+
+For example, if a token is saved in a file named `my-token`, it can be used to get the object `/ospool/PROTECTED/auth-test.txt` by running:
+
+```bash
+# Get the object with the path to token file provided via -t flag
+$ pelican object get -f https://osg-htc.org /ospool/PROTECTED/auth-test.txt downloaded-auth-test.txt -t my-token
+```
+
+## PUT an object to Your Federation
+Another powerful Pelican client command is the `pelican object put` command. This command does a simple PUT request to add your object to your federation. Putting files into a federation always requires a token therefore, for the example, we will need a token to perform these requests (see the [previous section](#get-a-protected-object-from-your-federation) for more information). Here is how you can use `pelican object put`:
+
+```bash
+pelican object put <path/to/local/file> pelican://<federation-url></namespace-prefix></path/to/file> -t </path/to/token/file>
+```
+
+> **Note:** you can also specify the federation url here with the `-f` flag, just be sure not to include it in the request URL as the host name if you decide to do so.
+
+## Pelican Object Copy
+
+> **Note**: We are phasing out the `object copy` command and we recommend user  use `object get` and `object put` command instead.
+
+As well with Pelican's `object get` and `object put` commands, there is also an older command called `pelican object copy`. This functions the same as the `pelican object get/put` except that it works for both gets and puts. For example, to do an `object get`:
+
+```bash
+pelican object copy pelican://<federation-url></namespace-prefix></path/to/file> <local/path/to/file>
+```
+
+and to do an `object put`:
+
+```bash
+pelican object copy <path/to/local/file> pelican://<federation-url></namespace-prefix></path/to/destination> -t </path/to/token/file>
+```
+
+## Utilizing Queries with your URL
+The Pelican client has the ability to select various amount of queries specified in the remote path to an object. The queries the current Pelican client accepts are: `?pack`, `?recursive`, and `?directread`.
```

Comment:
How about "Users can modify the behavior of requests through the Pelican client by passing URL query parameters in the remote path of an object. Currently supported queries include ..."